app/[locale]/ethereum-history-founder-and-ownership/page.tsx

```diff
@@ -0,0 +1,589 @@
+              </p>
+              <UnorderedList>
+                <ListItem>
+                  {t.rich(
+                    "page-ethereum-history-founder-and-ownership-when-did-ethereum-launch-description-2",
+                    {
+                      strong: Strong,
+                    }
+                  )}
+                </ListItem>
+                <ListItem>
+                  {t.rich(
+                    "page-ethereum-history-founder-and-ownership-when-did-ethereum-launch-description-3",
+                    {
+                      strong: Strong,
+                      announcement: (chunks) => (
+                        <Link href="https://www.youtube.com/watch?v=l9dpjN3Mwps">
+                          {chunks}
+                        </Link>
+                      ),
+                    }
+                  )}
+                </ListItem>
+                <ListItem>
+                  {t.rich(
+                    "page-ethereum-history-founder-and-ownership-when-did-ethereum-launch-description-4",
+                    {
+                      strong: Strong,
+                    }
+                  )}
+                </ListItem>
+                <ListItem>
+                  {t.rich(
+                    "page-ethereum-history-founder-and-ownership-when-did-ethereum-launch-description-5",
+                    {
+                      strong: Strong,
+                    }
+                  )}
+                </ListItem>
+                <ListItem>
+                  {t.rich(
+                    "page-ethereum-history-founder-and-ownership-when-did-ethereum-launch-description-6",
+                    {
+                      strong: Strong,
+                    }
+                  )}
+                </ListItem>
+                <ListItem>
+                  {t.rich(
+                    "page-ethereum-history-founder-and-ownership-when-did-ethereum-launch-description-7",
+                    {
+                      strong: Strong,
+                    }
+                  )}
+                </ListItem>
+              </UnorderedList>
+              <CommentCard
+                description={t(
+                  "page-ethereum-history-founder-and-ownership-when-did-ethereum-launch-description-8"
+                )}
+                name="Joseph Lubin"
+                title={t(
+                  "page-ethereum-history-founder-and-ownership-when-did-ethereum-launch-description-9"
+                )}
+              />
+              <p>
+                {t(
+                  "page-ethereum-history-founder-and-ownership-when-did-ethereum-launch-description-10"
+                )}
+              </p>
+              <p>
+                {t.rich(
+                  "page-ethereum-history-founder-and-ownership-when-did-ethereum-launch-description-11",
+                  {
+                    genesisBlock: (chunks) => (
+                      <Link href="https://etherscan.io/block/0">{chunks}</Link>
```

Comment:
open-source alternative
```suggestion
                      <Link href="https://eth.blockscout.com/block/0">{chunks}</Link>
```